### PR TITLE
Indirect Diffraction - Re-sizeable interface

### LINKED ISSUE #24921: Indirect Diffraction not Re-sizeable
### Expected behavior
The Indirect Diffraction interface should be re-sizeable as it is difficult to use it when on a laptop or other small-screen devices.

### Steps to reproduce the behavior
1.`Interfaces`->`Indirect`->`Diffraction`

### Platforms affected
All

### PR DESCRIPTION
**Pull request by robertapplin**
**Description of work.**
This PR ensures you can re-size the Indirect diffraction interface. This is useful as the diffraction interface was previously difficult to use on devices with smaller screens.

**To test:**
1.`Interfaces`->`Indirect`->`Diffraction`
2. Try to resize the interface by making it smaller

*No Release notes required I think as this is a minor change*

Fixes #24921

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py [hunk 1]
```diff
@@ -113,9 +122,13 @@ def validateInputs(self):
 
         # Validate input files
         input_files = self.getProperty('InputFiles').value
+
         if len(input_files) == 0:
```
Comment (TTitcombe) [inline Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py]: Should this not be part of the "invalid_number_range" function?

**Review by TTitcombe — CHANGES_REQUESTED** [review]
A question on code placement, plus possibility of unit tests

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py (hunk 1 above)
Comment (robertapplin) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py]: They are checking for different problems so they should be separate, but I see why you think this as the name I gave the function `contains_invalid_number_range` is not clear so I have changed it to `contains_non_ascending_range`

**Review by TTitcombe — APPROVED** [review]
Interface can now be resized :+1: There is a good minimum size so there is not danger of a user accidentally making it so small they lose track of the interface.

Good testing. I agree that it's probably too small a change to require release notes (unless lots of users have requested this as a feature)

**Pull request merged**